_posts/2020-02-27-react-conf-2020.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "ReactConf 2020"
+date:   2020-03-02
+tags: React Sydney
+author: Natalie Laing
+image: /img/blog/reactConf/react.png
+---
+
+## Introduction 
+React Conf AU is the first React conference in the southern hemisphere and it didn't disappoint. The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
```

Comment:
> The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
 
This sentence doesn't make sense. Perhaps it should be:

The quality of speakers was spot on and we got a chance to meet the faces behind the projects that make React what it is today.